homeworks/ovarikristof/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,85 @@
+ BEKÉRÜNK NEVEKET A FELHASZNÁLÓKHOZ:
+player_1 = input("Please enter a name for Player 1: ")
+player_2 = input("Please enter a name for Player 2: ")
+
+# BEKÉRJÜK, HOGY HÁNY KÖRT AKARNAK JÁTSZANI,
+#ÉS MEGHATÁROZZUK, HOGY CSAK PÁRATLAN KÖRÖK MEGADÁSA UTÁN INDUL A JÁTÉK
+def game_rounds():
+    while True:
+        try:
+            rounds_want_to_play = int(input("Please enter how many rounds do you want to play: "))
+            if rounds_want_to_play % 2 == 1:
+                return rounds_want_to_play
+            else:
+                print("Please enter an odd number, else there won't be a winner!")
+        except ValueError:
+            print("Please enter a valid number!")
+
+
+#MEGKÉRJÜK A JÁTÉKOSOKAT, HOGY ADJÁK MEG A VÁLASZAIKAT
+def get_players_1_choice(player_1):

```

Comment:
a get_players_1_choice() és a get_players_2_choice() gyakorlatilag ugyanazt csinálja, csak máshogy hívjuk a paramétert amit használnak. Próbáld meg kérlek úgy, hogy írsz egy általános function-t, ami csak bekér egy választ, és meg tudod hívni az 1-es és 2-es játékosra is:

```
def get_player_choice(player):
    while True:
        player_choice = input(f"{player}, please enter your choice (rock, paper, scissors): ").lower()
        if player_choice in ["rock", "paper", "scissors"]:
            return player_choice
        else:
            print("Please enter only the following: rock, paper, scissors!")
```

Amikor ezt meghívod, akkor ugyanúgy ahogy most, átadhatod neki a player_1-t és a player_2-t az 55. és 56. soron.